plugins/Toolbox/src/Toolbox.py

```diff
@@ -237,6 +249,11 @@ def _fetchPackageData(self):
         # Gather installed packages:
         self._updateInstalledModels()
 
+    def _fetchUserInstalledPlugins(self):
+        self._prepareNetworkManager()
```

Comment:
Nitpicking: The network manager only needs to be prepared if the user is logged in.